Localization/ko-KR/home.yaml

```diff
@@ -0,0 +1,20 @@
+--- 
+language: en-US
+text: 
```

Comment:
Your ``home.yaml`` is longer than the one in en-US, so they don't match up. Could you update your local branch to include your, please?

Reply:
Sure I will. I asked you through the Discord Translate channel that the en-US version of home.yaml does not include the whole texts compare to the actual webpage.

I will make sure to match the texts with en-US and make a request.

Reply:
Ah, I meant, could you upload your version of en-US yaml and your more complete version of ko-KR yaml, please? We want the more complete version of home.yaml for en-US and ko-KR. We will get all other languages to follow your home.yaml.